docs/components/motor/roboclaw.md

```diff
@@ -22,7 +22,8 @@ Follow [this guide](https://resources.basicmicro.com/roboclaw-motor-controllers-
 
 {{< /alert >}}
 
-To configure a `roboclaw` motor as a component of your machine:
+Make sure to physically connect your motor to your machine's computer and turn it on.
```

Comment:
This one I'm unsure about which is why I haven't suggested anything. Can you check?

Reply:
I don't have access to a roboclaw at the moment but I can ask rand.